src/solver/optimisation/hourly_csr_problem.h

```diff
@@ -40,5 +40,17 @@ class HOURLY_CSR_PROBLEM
     std::map<int, double> rhsAreaBalanceValues;
     std::set<int> varToBeSetToZeroIfBelowThreshold; // place inside only ENS and Spillage variable
     std::set<int> ensSet;                           // place inside only ENS inside adq-patch
-    std::set<int> linkSet; // place inside only links between to zones inside adq-patch
+    struct LinkVariable
+    {
+        LinkVariable() : directVar(-1), indirectVar(-1)
+        {
+        }
+        LinkVariable(int direct, int indirect) : directVar(direct), indirectVar(indirect)
+        {
+        }
+        int directVar;
```

Comment:
I suppose we need the 2 data members  **directVar** or **indirectVar**, and that only 
```C++
bool directVar;
``` 
would not have been sufficient

Reply:
No, for each alg. flux variable, we need the corresponding direct & indirect flux variables